Ne d bot scoring automations

nericksen opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

moishce commented on Packs/Base/Scripts/GetIndicatorDBotScoreFromCache/GetIndicatorDBotScoreFromCache.yml:
```diff
@@ -0,0 +1,20 @@
+subtype: python3
+tags: []
+type: python
+fromversion: 666666.0.0
+tests:
```
Fix the fromversion

nericksen replied:
Fixed now, thanks!

moishce reviewed (commented):
@nericksen Thanks for contributing.
Some fixes
- These scripts should be in the Common Scripts pack.
- Need to add a readme to both scripts

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=12309) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=12309) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: dantavori<br/>:white_check_mark: nericksen<br/>:x: tallieber<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=12309) it.</sub>

nericksen commented on Packs/Base/Scripts/GetIndicatorDBotScoreFromCache/GetIndicatorDBotScoreFromCache.py:
```diff
@@ -0,0 +1,38 @@
+        "HumanReadable": md
+    }
+
+    demisto.results(entry)
```
updated

nericksen commented on Packs/Base/Scripts/GetIndicatorDBotScoreFromCache/GetIndicatorDBotScoreFromCache.py:
```diff
@@ -0,0 +1,38 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+value = demisto.args().get("value", None)
```
added main function as requested

nericksen commented:
Scripts were moved to the `CommonScripts` from `Base` pack and README were generated.  There may be an issue with the branching structure although as best I can tell `demisto:contrib/nericksen_ne_DBotScoring-Automations` needs to be resynced with master. That should remove the miscellaneous commits from others that are now polluting this branch I think.  They kept appearing when I merged in the latest from master or tried to rebase. Let me know your thoughts @moishce

nericksen commented:
or should I open a new PR to resolve the branching issue?

ShahafBenYakir commented:
@nericksen No need for new PR you will just need to update you branch with `demisto:master`

nericksen commented:
@ShahafBenYakir `demisto:master` was merged into this branch.

nericksen commented:
@moishce perhaps we should merge this in as a new command and resolve the issue with the existing automation separately?

moishce commented on Packs/CommonScripts/Scripts/GetIndicatorDBotScoreFromCache/GetIndicatorDBotScoreFromCache.yml:
```diff
@@ -0,0 +1,20 @@
+args:
+- description: The single indicator value to check
+  name: value
```
```suggestion
  name: value
  required: true
```

nericksen replied:
Fixed, thanks!

nericksen commented:
Updated with most recent changes from main, implemented additional feedback.  We should be good to go here, but let me know if anything else is needed. Thanks for feedback and review!
